WalletWasabi/Tor/Http/TorHttpClient.cs

```diff
@@ -65,30 +66,41 @@ public async Task<HttpResponseMessage> SendAsync(HttpMethod method, string relat
 			request.Content = content;
 		}
 
-		return await SendAsync(request, token).ConfigureAwait(false);
+		return await SendAsync(request, cancellationToken).ConfigureAwait(false);
```

Comment:
Please stop these renamings. It makes it harder to review the PR and IMO it makes no sense.

Reply:
If for some reason you still want to do renamings, do it in a separate PR when renamings are kept separately.